framework/core/less/forum/NotificationList.less

```diff
@@ -2,7 +2,7 @@
   overflow: hidden;
 
   .App-primaryControl > button:not(:last-of-type) {
-    margin-right: 4px;
+    margin-right: 8px;
```

Comment:
I think even slightly more would be good.
```suggestion
    margin-right: 12px;
```

Reply:
![image](https://user-images.githubusercontent.com/38059171/188612058-83372a83-d71a-4969-9280-1d3af0ecd4d2.png)